modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/util/DDMIndexerImpl.java

```diff
@@ -359,6 +359,8 @@ public String extractIndexableAttributes(
 
 		Fields fields = toFields(ddmStructure, ddmFormValues);
 
+		boolean firstField = true;
```

Comment:
is the purpose of using this firstField variable just a source formatting thing, or are you actually trying to change the value that sb contains?

Reply:
i guess you are just trying to remove the trailing space